File: Python/10.summation_of_primes/rohitbindal.py
import math
n = 2000000


def isprime(val):
    i = 2
    while i <= int(math.sqrt(val)):
        if val % i == 0:
            # print("NP", val)
            return 0
        i += 1
    return 1


def main():
    sum_ = 2
    for i in range(3, n):
        if isprime(i):
            sum_ += i
    print(sum_)


if __name__ == "__main__":
    main()

File: CPP/10.summation_of_primes/rohitbindal.cpp
/*
Problem 10 - Summation of Primes
*/

#include <bits/stdc++.h>
using namespace std;

long n = 2000000;

int isPrime(long val) {
    for(auto i = 2;i<=sqrt(val);i++){
        if(val%i==0){
            return 0;
        }
    }
    return 1;
}

int main() {
    long sum=2;
    for(auto i = 3;i<n;i++){
        if(isPrime(i)){
            sum+=i;
        }
    }
    cout<<sum;
}
